kotlin/src/main/kotlin/org/jdbi/v3/core/kotlin/KotlinMapper.kt

```diff
@@ -38,22 +39,49 @@ import kotlin.reflect.jvm.jvmErasure
 
 private val nullValueRowMapper = RowMapper<Any?> { rs, ctx -> null }
 
-class KotlinMapper(clazz: Class<*>, private val prefix: String = "") : RowMapper<Any> {
-    private val kClass: KClass<*> = clazz.kotlin
+class KotlinMapper<T: Any>(clazz: Class<T>, private val prefix: String = "") : RowMapper<T> {
```

Comment:
Is this change considered {source, binary} compatible in the Kotlin world?  I'm not sure what the rules are.  I don't think it would be in Java land, but we probably just have to accept that the Kotlin world is more unstable for now?

Reply:
That's a great question, I'm not 100% sure.